docs/openlaw-object/README.md

```diff
@@ -0,0 +1,2141 @@
+};
+Openlaw.renderForReview(agreements[0].agreement, paragraphs);
+```
+
+**Response**
+
+Returns the HTML contents of the agreement as a string.
+
+Example
+
+```html
+<p class='no-section'>This Advisor Agreement is entered into between [[Company Name]] ("Corporation") and John Smith ("Advisor") as of September 19, 2018 ("Effective Date"). The parties agree as follows:</p><ul class='list-lvl-1'><li><p>1.  <strong>Services</strong>. Advisor agrees to consult with and advise Company from time to time, at Company's request (the "Services"). </p></li>
+```
+
+### renderForPreview
+
+Render a draft agreement for preview.
+
+```scala
+renderForPreview(
+  agreement: StructuredAgreement,
+  hiddenVariables: js.Array[String],
+  jsOverriddenParagraphs: js.Dictionary[String]
+): String
+```
+
+**Parameters**
+
+| Name | Type | Description |
+| ---- | ---- | ----------- |
+| `agreement` | [`StructuredAgreement`](#structuredagreement) | **Required.** A [`StructuredAgreement` object](#structuredagreement) representing the agreement to be rendered for preview. |
+| `hiddenVariables` | `Array<String>` | **Required.** An array of the agreement variables as strings. |
+| `jsOverriddenParagraphs` | `Object` | **Required.** An object containing the agreement paragraphs that were directly edited in draft mode. |
+
+Example
+
+```js
+// see examples above for #execute and #executeForReview for parameters
+const executionResult = Openlaw.execute(compiledTemplate.compiledTemplate, {}, params);
+const agreements = Openlaw.getAgreements(executionResult.executionResult);
+const hiddenVariables = ["Company Name", "Effective Date", "Number of Shares", "Years Vesting", "Unit of Vesting", "Company Signatory Email", "Advisor Name", "Company Signatory", "Advisor Email", "Time of Vesting", "No Services", "Advisor Address", "Company Address"];
+const paragraphs = {
+  0: "This Advisor Agreement is entered into between [[Company Name]] ("Corporation") and John Smith ("Advisor") as of September 19, 2018 ("Effective Date"). The parties agree as follows:"
+};
+Openlaw.renderForPreview(agreements[0].agreement, hiddenVariables, paragraphs);
+```
+
+**Response**
+
+Returns the HTML contents of the agreement as a string.
+
+Example
+
+```html
+<div class='openlaw-paragraph paragraph-1'><p class='no-section'>This Advisor Agreement is entered into between [[Company Name]] ("Corporation") and John Smith ("Advisor") as of September 19, 2018 ("Effective Date"). The parties agree as follows:</p></div><ul class='list-lvl-1'><li><div class='openlaw-paragraph paragraph-2'><p>1.  <strong>Services</strong>. Advisor agrees to consult with and advise Company from time to time, at Company's request (the "Services"). </p></div></li>
+```
+
+### parseMarkdown
+
+Parse selected paragraph text to edit (in draft mode) to HTML.
```

Comment:
Should we explain a bit more about the paragraphs? Is there anything relevant we can link to from the markup language docs?

Reply:
I added more of an explanation of editing paragraphs. There is no explanation of it in the markup language docs but I included a link to https://media.consensys.net/frictionless-contracting-628884915b8c which explains our draft tools and includes a demo video showing how to edit paragraphs directly.